common/include/pcl/console/time.h

```diff
@@ -55,19 +50,18 @@ namespace pcl
     {
       public:
 
-        TicToc () : tictic (), toctoc () {}
+        TicToc () : tictic () {}
 
         void 
         tic ()
         {
-          tictic = boost::posix_time::microsec_clock::local_time ();
+          tictic = std::chrono::steady_clock::now();
         };
 
         inline double 
         toc ()
         {
-          toctoc = boost::posix_time::microsec_clock::local_time ();
-          return (static_cast<double> ((toctoc - tictic).total_milliseconds ()));
+          return (static_cast<double> (std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::steady_clock::now() - tictic).count()));
```

Comment:
Do we need this static cast?

Reply:
`count()` does return an integral type during the return type is a floating type. So the compiler need to cast here. Without explicit cats we would get a compiler warning here, thats why I didn't removed the cast.

Reply:
Found an better way to get double value.